server/services/user-service.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
       console.log('existing user, no action required')
       return existingUser
     }
+    console.log('creating new user')

```

Comment:
The type returned from `user-service` is different depending on whether the user exists or is created, this need addressing in the future.

Reply:
Has this been raised somewhere trackable?

Reply:
No, we haven't decided on the location to track technical debt as far as I'm aware. 

My preference is to have git hub issues against the repository so they are close to the code, rather than in Jira where they are harder to find, but then there's less visibility from an organisational standpoint. 

For now I'll add cards to Jira.